tools/GenerateHelp.ps1

```diff
@@ -0,0 +1,50 @@
+#Requires -Modules platyPS
+
+[CmdletBinding()]
+Param(
+    [Parameter()]
+    [Switch]$ValidateMarkdownHelp,
+
```

Comment:
```<nit>```remove extra veritcal space.  You should only have a vertical space after a closing brace ```}```,  or the like ```</nit>```